learning_journal/learning_journal/templates/mytemplate.pt

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
If you are no longer using this template, you should remove it from your repository.